src/decoder/vorbis.rs

```diff
@@ -73,6 +74,14 @@ where
     fn total_duration(&self) -> Option<Duration> {
         None
     }
+
+    #[inline]
+    fn try_seek(&mut self, pos: Duration) -> Result<(), SeekNotSupported> {
+        // number of PCM samples per channel
+        let samples = pos.as_secs_f32() * self.sample_rate() as f32;
+        self.stream_reader.seek_absgp_pg(samples as u64).unwrap();
```

Comment:
Error should not be unwrapped but passed up. We are returning a result anyway, lets make use of it.